bundles/AdminBundle/EventListener/AdminAuthenticationDoubleCheckListener.php

```diff
@@ -17,7 +17,7 @@
 
 use Pimcore\Bundle\AdminBundle\Controller\DoubleAuthenticationControllerInterface;
 use Pimcore\Bundle\AdminBundle\EventListener\Traits\ControllerTypeTrait;
-use Pimcore\Bundle\AdminBundle\Security\User\TokenStorageUserResolver;
+use Pimcore\Security\User\TokenStorageUserResolver;
 use Pimcore\Bundle\CoreBundle\EventListener\Traits\PimcoreContextAwareTrait;
 use Pimcore\Http\Request\Resolver\PimcoreContextResolver;
```

Comment:
Even it is `@internal` we should still add `@deprecated`, just for reference I think, as it is getting removed in Pimcore 11.